### ISSUE TITLE
Can't parse json containing very large number

### ISSUE THREAD

**Issue by chvck** [couchbaselabs/couchbase-shell#188]
```
 {
            "cmd_get": 1.998001998001998,
            "couch_docs_actual_disk_size": 29044032244,
            "couch_docs_data_size": 20956132290,
            "couch_spatial_data_size": 0,
            "couch_spatial_disk_size": 0,
            "couch_views_actual_disk_size": 18785057365,
            "couch_views_data_size": 5323749123,
            "curr_items": 18446744073715094000,
            "curr_items_tot": 16626891,
            "ep_bg_fetched": 0,
            "get_hits": 0.999000999000999,
            "mem_used": 14482854736,
            "ops": 1.998001998001998,
            "vb_active_num_non_resident": 2640298,
            "vb_replica_curr_items": 11086582
          }
```
The `curr_items` field of this doc causes nushell to panic with an overflow as it's trying to parse this as a u64. Will likely need to chat to the nu folks about the best way to get this sorted.

**Comment by chvck** [comment]
See https://github.com/nushell/nushell/issues/5907

**Comment by RekhaDS** [comment]
Hello Charlie - @chvck - looks like this issue got fixed in nushell
https://github.com/nushell/nushell/pull/6096

Looks like the latest binary on the doc site https://couchbase.sh/docs/ does not have the fix.

open temp.json | wrap content | doc insert
thread 'main' panicked at 'Internal error: json parsing: ParseIntError { kind: PosOverflow }', /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/nu-json-0.63.0/src/util.rs:220:52
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by Westwooo** [comment]
This does appear fixed by the bump to nushell 0.90. I saved the doc supplied to `testing.json` the upserted as follows: 
```
👤 Administrator 🏠 local in 🗄 default
> open testing.json | doc upsert testing $in
```
This results in the following being saved to the database:
```
{
  "cmd_get": 1.998001998001998,
  "couch_docs_actual_disk_size": 29044032244,
  "couch_docs_data_size": 20956132290,
  "couch_spatial_data_size": 0,
  "couch_spatial_disk_size": 0,
  "couch_views_actual_disk_size": 18785057365,
  "couch_views_data_size": 5323749123,
  "curr_items": 1.8446744073715094e19,
  "curr_items_tot": 16626891,
  "ep_bg_fetched": 0,
  "get_hits": 0.999000999000999,
  "mem_used": 14482854736,
  "ops": 1.998001998001998,
  "vb_active_num_non_resident": 2640298,
  "vb_replica_curr_items": 11086582
}
```
